routes/jobs.js

```diff
@@ -0,0 +1,14 @@
+
+const express = require('express');
+const router = express.Router();
+const { createJob, getAllJobs, getJobById, updateJob, deleteJob } = require('../controllers/jobs');
+const { authenticateJWT } = require('../middleware/auth');
+
+// Protect task routes with authentication
+router.post('/job', authenticateJWT, createJob);
+router.get('/jobs', authenticateJWT, getAllJobs);
+router.get('/job/:id', authenticateJWT, getJobById);
+router.patch('/job/:id', authenticateJWT, updateJob);
+router.delete('/job/:id', authenticateJWT, deleteJob);
+
```

Comment:
When designing REST APIs, we don't change between singular and plural depending on the operation, it's best practice to stick with one way and let HTTP methods (GET, POST...) define what the endpoint returns.

```suggestion
// Protect task routes with authentication
router.post('/jobs', authenticateJWT, createJob);
router.get('/jobs', authenticateJWT, getAllJobs);
router.get('/jobs/:id', authenticateJWT, getJobById);
router.patch('/jobs/:id', authenticateJWT, updateJob);
router.delete('/jobs/:id', authenticateJWT, deleteJob);

```

Reply:
https://swagger.io/resources/articles/best-practices-in-api-design/